lib/cobra_commander/cli.rb

```diff
@@ -10,6 +10,13 @@ def ls(app_path)
       puts FormattedOutput.new(app_path).run!
     end
 
+    desc "count_dependencies APP_PATH", "Outputs count of components in APP_PATH dependent on COMPONENT"
+    method_option :component, required: true, aliases: "-c", desc: "Name of component: core_models or nitro_component_transition"
+    method_option :format, default: "count", aliases: "-f", desc: "count or list"
```

Comment:
`count_dependencies -f list` sounds a bit funky. Any alternatives?

Reply:
James Brown is funky. Not sure specifically what you're referring to. `count_dependencies` or `-f list` or what. I'm open to renaming. I had a 2 second thought to use `cd` but that is stuck in my mind permanently as *change directory*.

Reply:
Just the combination of the verbs `count` and `list` in the same command.

Reply:
What about `dependencies_of -f count foobar` or `dependencies_of -f list foobar`?

Reply:
I'm good with that.